index.html

```diff
@@ -76,6 +76,71 @@
         </section>
         </div>
         <footer>
+          <div class="map-wrapper">
+            <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2452.7280006260867!2d21.48412931521773!3d52.066476177492945!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x4718d2fa42024705%3A0xf4248a53ec6c2b01!2sS%C5%82oneczna+8A%2C+05-340+Ko%C5%82biel!5e0!3m2!1spl!2spl!4v1451830097052" allowfullscreen></iframe>
+          </div>
+          <div class="contact-wrapper">
+            <ul class="contact-list">
+              <li>
+                <ul>
+                  <li class="bargar-wrapper">
+                    <img src="/img/barber_map_icon.png" alt="barber">
+                  </li>
+                  <li class="bargar-wrapper">
+                    <ul class="bargar-list">
+                      BARBER
+                      <li>
+                        poniedziałek-piątek
+                        <br>
+                        sobota
+                      </li>
+                      <li class="adress-list">
+                        ul.Grzybowska 66/6
+                        <br>
+                        02-300 Warszawa
+                      </li>
+                      <li class="tel-list">
+                        tel (+48)555 555 555
+                        <br>
+                        kontakt@barbergarage.pl
+                      </li>
+
+                    </ul>
+                </li>
+              </ul>
+            </li>
+            <li class>
+              <img src="/img/map-line.png" alt="to jest oddzielająca linia">
+            </li>
+            <li>
+                <ul>
+                  <li class="bargar-wrapper">
+                    <img src="/img/garage_map_icon.png" alt="garage">
+                  </li>
+                  <li class="bargar-wrapper">
```

Comment:
wtf is `bargar`? 
![image](https://cloud.githubusercontent.com/assets/980955/12079394/ca98b002-b237-11e5-9942-39a5248dc68d.png)